docs/docs/dev/security/certificates.md

```diff
@@ -122,6 +122,46 @@ You can also provide the CA via environment variable `KUMA_CONTROL_PLANE_CA_CERT
 
 See [Data plane proxy authentication](./dp-auth.md) and [Zone Ingress authentication](./zone-ingress-auth.md).
 
+## Prometheus to control plane communication
+
+It is possible to enabled TLS on the Monitoring Assignment Discovery Service.
+By default it will use the certificate used in `tls-general` but TLS will be disabled.
+You can enable it by using either the environment variable `KUMA_MONITORING_ASSIGNMENT_SERVER_TLS_ENABLED=true` or the yaml config `monitoringAssignmentServer.tlsEnabled=true`.
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "Kubernetes (kumactl)"
```

Comment:
If this is for 2.0, I'm not sure if these tab options will work in Jekyll. Here's a doc on how we format tabs on a page in Jekyll: https://docs.konghq.com/contributing/markdown-rules/#tabs